pkg/dbmate/postgres.go

```diff
@@ -174,6 +174,33 @@ func (drv PostgresDriver) DeleteMigration(db Transaction, version string) error
 	return err
 }
 
+// The sum of all charCodes in the string "dbmate" :)
+const advisoryLockKey = 621
```

Comment:
hmm, this could cause weird behavior for some applications if we don't have any way to namespace it...

Reply:
Yes this is true, I'm unsure how to tackle that problem, its hard to namespace since [postgres only accepts numbers](https://www.postgresql.org/docs/9.6/functions-admin.html#FUNCTIONS-ADVISORY-LOCKS) as far as i can see. We could either set this to something bigger and more random i guess, but it still feels shaky. Or we could namespace it further by setting two numbers, which effectively gives the same result, but assumingly with less collision risk? Maybe if this happens to collide with an application's lock key we can always give the user the option to specify it themselves - on the other hand as a user tracking that down as the solution to such a problem seems... hard. Idk, how much of a dealbreaker is this do you think?